internal/server/server.go

```diff
@@ -43,6 +44,7 @@ func CreateServerCommand(logFn LogFunc) *cobra.Command {
 	fset.String("db-addr", "", "the TCP host and port of the Perseus DB")
 	fset.String("db-user", "", "the login to be used when connecting to the Perseus DB")
 	fset.String("db-pass", "", "the password to be used when connecting to the Perseus DB")
+	fset.String("db-name", "perseus", "the name of the Perseus DB to connect to")
```

Comment:
Should `perseus` be a const here, so this and `withDBName()` always have the same default, or am I being too nit-picky?

Reply:
Probably okay to not worry about it, but a reasonable suggestion still.  Will update.